src/Uno.Templates/content/unoapp/GitHub/workflows/ci.yml

```diff
@@ -0,0 +1,84 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - $DefaultBranchName$
+      - release/**
+
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches:
+      - $DefaultBranchName$
+      - release/**
+env:
+  STEP_TIMEOUT: 60
+
+jobs:
+  smoke_test:
+    name: Smoke Test
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Dependencies
+        timeout-minutes: ${{ fromJSON(env.STEP_TIMEOUT) }}
```

Comment:
Is the whole "fromJSON" required for this? Can't we specify STEP_TIMEOUT as an int variable

Reply:
See the referenced thread... `timeout-minutes` expects a thread but gets a string from `${{ env.VARIABLE_NAME }}` where the `fromJson(...)` is able to properly convert it to the expected int type...
https://github.com/orgs/community/discussions/26332#discussioncomment-3251517